website/package-lock.json

```
@@ -14095,7 +14095,6 @@
       "version": "3.3.1",
       "resolved": "https://registry.npmjs.org/react-player/-/react-player-3.3.1.tgz",
       "integrity": "sha512-wE/xLloneXZ1keelFCaNeIFVNUp4/7YoUjfHjwF945aQzsbDKiIB0LQuCchGL+la0Y1IybxnR0R6Cm3AiqInMw==",
```

Let's revert this change.

```suggestion
      "integrity": "sha512-wE/xLloneXZ1keelFCaNeIFVNUp4/7YoUjfHjwF945aQzsbDKiIB0LQuCchGL+la0Y1IybxnR0R6Cm3AiqInMw==",
      "license": "MIT",
```